docs/source/visualisation/preview_datasets.md

```diff
@@ -0,0 +1,84 @@
+# Preview data in Kedro-Viz
+
+This page describes how to preview data from different datasets in a Kedro project with Kedro-Viz. Kedro-viz currently supports preview of `CSVDatasets` and `ExcelDatasets`
```

Comment:
> This page describes how to preview data from different datasets in a Kedro project with Kedro-Viz. Kedro-viz currently supports preview of `CSVDatasets` and `ExcelDatasets`

Please can you say what "current" is by saying something like: "Dataset preview was first introduced in Kedro-Viz version <whatever>, which offers preview for `CSVDatasets` and `ExcelDatasets`."